NEWS

```diff
@@ -2,6 +2,25 @@
 
 Sandia OpenSHMEM NEWS -- history of user-visible changes.
 
+v1.5.3rc1
+---------
+- Added several enhancements to better support SOS as a backend for Intel® SHMEM.
+- Added extension support for GPU RDMA and external heap creation.
+- Added support for multi-NIC configurations via libfabric. The feature is
+  enabled by default. It can be disabled with the environment variable,
+  SHMEM_OFI_DISABLE_MULTIRAIL=1.
+- Added initial support for multi-NIC topology optimizations via hwloc.
+- Moved the "tests-sos" package of unit tests and performance benchmarks to a
+  new Git submodule hosted at https://github.com/openshmem-org/tests-sos.
+- Added shmemx_ibput and shmemx_ibget as extension APIs.
+- Added shmemx_signal_add and shmemx_signal_set as extension APIs.
+- Added several configuration flags to optimize for the CXI libfabric provider:
+  --enable-ofi-inject and --enable-nonfetch-amo, which are enabled by default.
+- Manpage generation is now disabled by default to shorten build times. It can
+  be re-enabled during configuration with the --enable-manpages flag.
+- Multiple bugfixes, including in teams configuration, remote-virtual-addressing
```

Comment:
`Included multiple bugfixes`? minor change to make the bullets similar.

Reply:
Ah, good call - added in https://github.com/Sandia-OpenSHMEM/SOS/pull/1142/commits/dd5190009f2950390e4ae6f54588642eae402a76